packages/core/test/unit/query-generator/rename-table-query.test.ts

```diff
@@ -1,15 +1,18 @@
 import { createSequelizeInstance, expectsql, sequelize } from '../../support';
 
 const dialect = sequelize.dialect;
+const changeSchemaNotSetError = new Error('To move a table between schemas, you must set `options.changeSchema` to true.');
+const moveSchemaNotSupportedError = new Error(`Moving tables between schemas is not supported by ${dialect.name} dialect.`);
```

Comment:
I think I'm missing a test that moves a table across schemas without a rename

Reply:
It did exist, I've renamed it now.

Reply:
I was looking for a test like
`queryGenerator.renameTableQuery({ tableName: 'oldTable', schema: 'oldSchema' }, { tableName: 'oldTable', schema: 'newSchema' }, { changeSchema: true })` (note that tableName is the same, only schema has changed)

Does that also exist?

Reply:
I've added a new test now.
LMK if this is what you are looking for.